### PR TITLE
setup: add two missing requirements

### PR DESCRIPTION
**Pull request by sebix**
matplotlib is required in the tests since the last version 0.5.0 and mock was also missing

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ def get_version(module='spyder_kernels'):
                 'pyzmq>=17',
                 'jupyter-client>=5.2.3',
                 'cloudpickle',
-                'wurlitzer;platform_system!="Windows"']
+                'wurlitzer;platform_system!="Windows"',
+                'mock',
+                'matplotlib']
```
Comment (ccordoba12) [inline setup.py]: These are not package requirements, but test ones. So what you need to do is add an `extras_require` kwarg to `setup` below and list all our test requirements there (not just these two).

Our test requirements are listed here:

https://github.com/spyder-ide/spyder-kernels/blob/master/requirements/tests.txt

**Comment by ccordoba12** [comment]
I'm closing this because what's done against master doesn't affect our 0.x branch (where 0.5.0 was tagged).

I'll ask one of our contributors to fix this for you. Thanks for understanding.

**Comment by sebix** [comment]
It would have been possible to change the target branch...

**Comment by ccordoba12** [comment]
Sure, but you wouldn't be able to use `requirements/tests.txt` because it's not present in 0.x

**Pull request closed without merging**